neuralop/models/uno.py

```diff
@@ -225,6 +231,7 @@ def forward(self, x):
 
         if self.domain_padding is not None:
             x = self.domain_padding.pad(x)
+        print(x.shape)
```

Comment:
Let's remove this and the next print too

Reply:
done